docs/platform/tizen-rt/public-api/index.html

```diff
@@ -110,8 +110,8 @@
 <p>For Additional information of Tizen RT, please see below Documentation.</p>
 <h3>Documentation</h3>
 <ul>
-<li>Tizen RT Getting Started <a href="https://source.tizen.org/documentation/tizen-rt/tizen-rt-getting-started">page</a></li>
-<li>Tizen RT 1.0 Specification <a href="https://source.tizen.org/documentation/tizen-rt/tizen-rt-1.0-specification">page</a> </li>
+<li>Tizen RT Getting Started <a href="/platform/tizen-rt/rt-getting-started">page</a></li>
```

Comment:
I'm not sure with update as the hierarchy doesn't match with folder structure. Would you please check?